wordle-web/src/views/WordleView.vue

```diff
@@ -35,21 +37,22 @@ function checkGuess() {
 }
```

Comment:
submitGuess() should only run when the input letters length equal to the size of the words